gpytorch/lazy/kronecker_product_lazy_tensor.py

```diff
@@ -138,6 +138,11 @@ def diag(self):
                 raise RuntimeError("Diag works on square matrices (or batches)")
         return _kron_diag(*self.lazy_tensors)
 
+    def diagonalization(self, method: Optional[str] = None):
+        if method is None:
+            method = "symeig"
+        return super().diagonalization(method=method)
```

Comment:
@wjmaddox I think this is the right thing to do as a default for Kronecker, what do you think?

Reply:
confirmed offline that yes